quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java

```diff
@@ -234,6 +251,25 @@ private static Level toLevel(String categoryLevel) throws IllegalArgumentExcepti
         return LogContext.getLogContext().getLevelForName(categoryLevel.toUpperCase(Locale.ROOT));
     }
 
+    private static Optional<String> transformSyslogMaxLength(Optional<String> value, ConfigSourceInterceptorContext context) {
+        if (value.isPresent()) {
+            return value;
+        }
+
+        var syslogType = Configuration.getOptionalKcValue(LoggingOptions.LOG_SYSLOG_TYPE)
+                .map(String::toUpperCase)
+                .map(SyslogHandler.SyslogType::valueOf)
+                .orElseThrow(() -> new IllegalArgumentException("You need to provide a valid Syslog type"));
+
+        return Optional.of(
+                switch (syslogType) {
+                    case RFC5424:
+                        yield "2048";
+                    case RFC3164:
+                        yield "1024";
+                });
+    }
```

Comment:
Just out of curiosity. Do we really need this logic? Isn't it handled by Quarkus?